test/CMakeLists.txt

```diff
@@ -368,7 +370,9 @@ if (TARGET TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_tick_count DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_allocators DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_arena_priorities DEPENDENCIES TBB::tbb)
-    tbb_add_test(SUBDIR tbb NAME test_dynamic_link DEPENDENCIES TBB::tbb)
+    if (NOT TBB_EMSCRIPTEN)    
+        tbb_add_test(SUBDIR tbb NAME test_dynamic_link DEPENDENCIES TBB::tbb)
+    endif()
```

Comment:
I think the tests are already protected by `__TBB_DYNAMIC_LOAD_ENABLED`, so hopefully you don't need to test for `TBB_EMSCRIPTEN` here. And likewise for other tests in this file that are currently being avoided under Emscripten.